### ISSUE TITLE
Case Investigation - Recording issue via Labrus RTSP Camera

### ISSUE THREAD

**Issue by yashtandon113** [ant-media/Ant-Media-Server#5739]
### Short description
Their are some RTSP feeds published on a server via Labrus RTSP cameras. Their is no issue with the live stream, but recording for this specific camera is not fine. Sometimes size is in gigabytes, but recording is only a few seconds, and sometimes recording duration shows more than the original duration.

### Environment
* Ant Media Server version: v2.6.0 and above

### Steps to reproduce
1. Enable the mp4 recording on the server
2. Pull RTSP camera feed from Labrus camera and record for at least 1 hour
3. Stop the stream and check the recording.

### Expected behavior
Recording should be fine.

### Actual behavior
Recording is not fine, as described in the issue description.

**Comment by lastpeony** [comment]
In my initial test, I recorded a broadcast that lasted for 1 hour and 15 minutes, broadcast speed was consistent approximately 1x. The end recording was fine.

During the second test, I recorded for approximately 4 hours. In the first 1 hour and 45 minutes, the broadcast maintained a steady speed of around 1x. However, after this initial period, the broadcast speed decreased to a constant 0.22x and remained at this rate for about 2 hours. Subsequently, the speed dropped further to a constant 0.01x, and after roughly 15 minutes at this rate, I decided to stop the recording.

The recording quality was satisfactory during the period when the broadcast speed was around 1x. However, issues with the recording became apparent when the broadcast speed dropped to 0.22x and 0.01x. Notably, in the last second of the recording, there was an unusual acceleration, causing the final segment to extend beyond a typical second in duration. To summarize, on video player last second takes more than a second and video speeds up.

related or same problem with:
https://github.com/ant-media/Ant-Media-Server/issues/4838
